docs/prerequisites/systemconfiguration.md

```diff
@@ -6,68 +6,41 @@ weight: 100
 
-We require the following modules to be loaded:
-
-* `target_core_mod`
-* `tcm_loop`
-* `configfs`
-* `target_core_user`
-* `uio`
-
-> ⚠️ Other applications utilising [TCMU](http://linux-iscsi.org/wiki/LIO) cannot be run concurrently with Ondat. Doing so may result in corruption of data. On startup, Ondat will detect if other applications are using TCMU.
-
-> ⚠️ TCMU can be disabled using the [DISABLE_TCMU](/docs/reference/cluster-operator/configuration) StorageOSCluster spec parameter.
-
-Depending on the distribution, the modules are shipped as part of the base kernel package or as part of a kernel extras package which needs to be installed.
-
 ## Distribution Specifics
 
-The following distributions are supported by default:
-
-* RHEL 7.5
-* CentOS 7
-* Debian 9
-* Ubuntu Azure
-* RancherOS - Note CSI is not supported on RancherOS
-
-Ubuntu 16.04/18.04 requires the installation of additional packages.
+Current (non-EOL) versions of the following distributions are supported by default:
 
-> ⚠️ Ubuntu 16.04/18.04 AWS and Ubuntu 18.04 GCE do not provide the necessary linux-image-extra package - [see below](/docs/prerequisites/systemconfiguration#ubuntu-with-aws-or-gce-kernels) for more information
+* SUSE Linux Enterprise Server
+* Red Hat Enterprise Linux
+* CentOS
+* Debian
+* Ubuntu
 
-## Ubuntu Package Installation
+The following distributions include the prerequisite modules but are not yet tested exhaustively by the Ondat team:
 
-**Ubuntu 16.04/18.04 Generic** and **Ubuntu 16.04 GCE** require extra packages:
+* Bottlerocket
+* Google ContainerOS
 
-Ubuntu 16.04:
+The following distributions are currently not supported:
 
-```bash
-sudo apt -y update
-sudo apt -y install linux-image-extra-$(uname -r)
-```
+* Amazon Linux (lacks `target_core_mod` and `target_core_user`)
+* RancherOS (CSI is not supported on RancherOS)
 
-Ubuntu 18.04+:
+> 💡 If you require help with a specific issue with a listed distribution, [raise an issue on GitHub](https://github.com/ondat/documentation/issues) or reach out to us on our [Community Slack](https://slack.storageos.com)
 
-```bash
-sudo apt -y update
-sudo apt -y install linux-modules-extra-$(uname -r)
-```
+## Kernel Modules
 
-## Ubuntu With AWS Or GCE Kernels
-
-**Ubuntu 16.04/18.04 AWS** and **Ubuntu 18.04 GCE** do not yet provide the
-linux-image-extra package. As such you should either use **Debian**, **CentOS**
-or **RHEL**, or install the non-cloud-provider optimised Ubuntu kernel.
+We require the following modules to be loaded:
 
-> ⚠️ Installing the non-cloud-provider optimised Ubuntu kernel is something that should only be done with full understanding of potential ramifications.
+* `target_core_mod`
+* `tcm_loop`
+* `configfs`
+* `target_core_user`
+* `uio`
 
-```bash
-sudo apt -y update
```

Comment:
I see why we are removing this section, however there is no indication in the whole document on how to install those kmods in case they are using an ubuntu image that don't have them. Listing what packages can be installed in case of need is IMO useful.

Reply:
I've added a note explaining how to discover whether the user is missing any modules and how to install them if they are missing on older Ubuntu versions! Thanks for the feedback 🙌🏻